masonry_core/src/lib.rs

```diff
@@ -39,8 +39,17 @@
 //! Note that Masonry Core is currently focused primarily on the main Masonry crate itself, as we're
 //! not aware of any projects using Masonry Core as described in this paragraph.
 //!
+//! # Feature flags
+//!
+//! The following crate [feature flags](https://doc.rust-lang.org/cargo/reference/features.html#dependency-features) are available:
+//!
+//! - `default`: Enables the default features of [`vello`].
+//! - `tracy`: Enables creating output for the [Tracy](https://github.com/wolfpld/tracy) profiler using [`tracing-tracy`][tracing_tracy].
+//!   This can be used by installing Tracy and connecting to a Masonry with this feature enabled.
+//!
 //! [Masonry's documentation]: https://docs.rs/masonry/latest/
 //! [Masonry Winit]: https://docs.rs/masonry_winit/latest/
+//! [tracing_tracy]: https://crates.io/crates/tracing-tracy
```

Comment:
(The reason the same thing doesn't apply to `tracing_tracy` here is not because we shouldn't handle it the same way; instead it's much less important, as it's very unlikely that anyone would want to explore the docs of that library from this crate, so the homepage is as appropriate)